runtime/sema/checker.go

```diff
@@ -235,6 +236,16 @@ func WithPositionInfoEnabled(enabled bool) Option {
 	}
 }
 
+// WithLintingEnabled returns a checker option which enables/disables
+// advanced linting.
+//
+func WithLintingEnabled(enabled bool) Option {
```

Comment:
Must also be propagated to sub-interpreters in `NewSubInterpreter`

Reply:
This flag is for the checker. I think we don't need to pass it down?